### PR TITLE
Changed string concatenation to f-strings

### PR DESCRIPTION
**Pull request by cryptikkUS**

### CODE REVIEW

File: scratchclient/CloudConnection.py [hunk 1]
```diff
@@ -69,7 +69,7 @@ def set_cloud_variable(self, variable, value):
                 packet = {
                     "method": "set",
                     "name": (
-                        "☁ " + variable if not variable.startswith("☁ ") else variable
+                        f"☁ {variable}" if not variable.startswith("☁ ") else variable
```
Comment (Quantum-Codes) [inline scratchclient/CloudConnection.py:72]: This is wrong... you completely forgot about the if loop

File: scratchclient/Project.py [hunk 2]
```diff
@@ -37,23 +37,15 @@ def __init__(self, data, client):
...
             "X-Token": self._client.token,
             "x-requested-with": "XMLHttpRequest",
-            "Cookie": "scratchcsrftoken="
-            + self._client.csrf_token
-            + ";scratchlanguage=en;scratchsessionsid="
-            + self._client.session_id
-            + ";",
-            "referer": "https://scratch.mit.edu/projects/" + str(self.id) + "/",
+            "Cookie": f"scratchcsrftoken={self._client.csrf_token};scratchlanguage=en;scratchsessionsid={self._client.session_id};",
+            "referer": f"https://scratch.mit.edu/projects/{str(self.id)}/",
```
Comment (Quantum-Codes) [inline scratchclient/Project.py:48]: F string autoconvert to str

File: scratchclient/Project.py [hunk 3]
```diff
@@ -63,55 +55,37 @@ def _to_project_comment(self, data):
...
     def get_comment(self, comment_id):
         data = requests.get(
-            "https://api.scratch.mit.edu/users/"
-            + self.author.username
-            + "/projects/"
-            + str(self.id)
-            + "/comments/"
-            + str(comment_id)
-            + "/"
+            f"https://api.scratch.mit.edu/users/{self.author.username}/projects/{str(self.id)}/comments/{str(comment_id)}/"
```
Comment (Quantum-Codes) [inline scratchclient/Project.py:58]: Same here

**Review by Quantum-Codes — CHANGES_REQUESTED** [review]
Just change all str conversion everywhere. 
Although I'm not sure whether cubey looks at this repo anymore or not

**Comment by cryptikkUS** [comment]
Corrected

**Comment by CubeyTheCube** [comment]
Thanks for the contribution, but I'm planning to completely refactor the code in this library so this PR wouldn't be of much use.

**Pull request closed without merging**